### DISCUSSION TITLE
Skaffold for local development?

### DISCUSSION THREAD

**Discussion by rocketproto** [authelia/authelia#6200] [Q&A] [answered]
Hey! Just wondering if anyone has experimented with [Skaffold](https://skaffold.dev/) while developing inside a Kubernetes environment or am I diving into uncharted territory? Looking at getting it built and running in a dev environment so I can give #2319 a fair go. Still fairly new to OS contributions so forgive me if I get any of the processes wrong :)

I've got my own Kubernetes manifests rather than using the helm chart (couldn't get it working, bit of a skill issue at the time and haven't revisited for half a year). I've setup skaffold in plenty of repos that just use kube manifests, but never tried with helmcharts. The environment is a k3s cluster running istio with extAuth. 

If I was able to get Skaffold up and running, would there be appetite for it to be committed back to the main or chart repo?

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not getting back to you sooner. We do not use Skaffold for local development and have not experimented with it. We have our own system which uses docker compose and some fancy hot reload tech.See here: https://www.authelia.com/contributing/development/build-and-test/
